plugin/kubernetes/setup.go

```diff
@@ -196,6 +196,15 @@ func ParseStanza(c *caddy.Controller) (*Kubernetes, error) {
 			args := c.RemainingArgs()
 			if len(args) > 0 {
 				k8s.APIServerList = args
+				if len(args) > 1 {
+					// If multiple endoints specified, then only http allowed
+					for i := range args {
+						parts := strings.SplitN(args[i], "://", 2)
```

Comment:
you can just use url.Parse and check the scheme, seems slightly nicer than using your own "parser"